include/swift/Frontend/ArgsToFrontendInputsConverter.h

```diff
@@ -0,0 +1,78 @@
+//===--- ArgsToFrontendInputsConverter.h ------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_FRONTEND_ARGSTOFRONTENDINPUTSCONVERTER_H
+#define SWIFT_FRONTEND_ARGSTOFRONTENDINPUTSCONVERTER_H
+
+#include "swift/AST/DiagnosticConsumer.h"
+#include "swift/AST/DiagnosticEngine.h"
+#include "swift/Frontend/FrontendOptions.h"
+#include "llvm/Option/ArgList.h"
+
+using namespace swift;
+using namespace llvm::opt;
```

Comment:
Oops, sorry. [No `using namespace` in headers](http://llvm.org/docs/CodingStandards.html#do-not-use-using-namespace-std), please.